pkgs/by-name/pi/pie-cli/package.nix

```
@@ -0,0 +1,26 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pie-cli";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Falconerd";
+    repo = "pie";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-qJaQyet6pjvPhBg6p0wxSIJtZE+P7A7XVqzAnvGn12E=";
+  };
+
+  buildPhase = "cc -o pie pie_cli/pie.c -lm";
+  installPhase = "install -Dm 755 pie $out/bin/pie";
+
+  meta = with lib; {
+    maintainers = with maintainers; [ sigmanificient ];
+    homepage = "https://github.com/Falconerd/pie";
+    license = lib.licenses.mit;
+    platforms = lib.platforms.linux;
```

```suggestion
    homepage = "https://github.com/Falconerd/pie";
    maintainers = with lib.maintainers; [ sigmanificient ];
    license = lib.licenses.mit;
    platforms = lib.platforms.linux;
```

Add a short `description`.